src/lib/features/NavMenu.stories.svelte

```diff
@@ -16,32 +16,35 @@
   const { Story } = defineMeta({ title: 'UI Components/Features/NavMenu', component: NavMenu });
 </script>
 
-<Story name="Desktop" id="navMenu">
-  <div class="sticky top-0 bg-white">
-    <NavMenu {menuItems} />
-    <p class="xs:block lg:hidden">*** If you are not seeing the menu, try increasing the window size.</p>
-  </div>
-
-  <p class="block">You may need to have a shorter browser to test the highlight moving</p>
-
-  <div id="mission" class="bg-red-900 py-96">
-    <h2>Mission</h2>
-  </div>
-  <div id="about" class="bg-blue-900 py-96">
-    <h2>About</h2>
-  </div>
-  <div id="other" class="bg-green-900 py-96">
-    <h2>Other</h2>
-  </div>
-  <div id="user" class="bg-red-900 py-96">
-    <h2>User</h2>
-  </div>
-
-  <div id="developers" class="bg-blue-900 py-96">
-    <h2>Developers</h2>
-  </div>
-
-  <div id="partners" class="bg-green-900 py-96">
-    <h2>Partners</h2>
-  </div>
+<p class="xs:block lg:hidden">*** If you are not seeing the menu, try increasing the window size.</p>
+
+<Story name="Desktop" id="navMenu" args={{ ...NavMenu.props, menuItems }}>
+  {#snippet children(args)}
+    <NavMenu {...args} />
+  {/snippet}
 </Story>
+
+<!-- This section will not render unless outside of the storytags, which in turn does not
+   have the desired effect. It should be inside of a sticky block (removed). -->
```

Comment:
If we dropped the sticky block, do we need to also just remove these sections? The only way they work is if there is a sticky on the outside of the header.